native/packages/react-native-bpk-component-badge/README.md

```diff
@@ -44,8 +45,10 @@ export default class App extends Component {
 ```
 ## Props
 
-| Property              | PropType                                                                 | Required | Default Value |
+| Property              | PropType                                                                  | Required | Default Value |
 | --------------------- | --------------------------------------------------------------------------| -------- | ------------- |
 | message               | string                                                                    | true     | -             |
 | docked                | oneOf('start', 'end')                                                     | false    | null          |
+| iconSeparator         | string                                                                    | false    | null          |
+| icons                 | arrayOf(string)                                                           | false    | []            |
```

Comment:
Can we make this more clear that it needs to be an icon string specifically? `arrayOf(BpkIcon.icons)` or something?